code/frameworks/html-vite/src/index.ts

```diff
@@ -0,0 +1,8 @@
+// exports for builder-vite
+export { createChannel as createPostMessageChannel } from '@storybook/channel-postmessage';
```

Comment:
Looks like we don't need any of these exports anymore except the `StorybookConfig` type. @IanVS any other changes we need to make to get this working?

Reply:
Nope, you're right, we can remove the extra re-exports now.  :+1: